Source/WebCore/dom/Traversal.h

```diff
@@ -43,9 +43,22 @@ class NodeIteratorBase {
 
 protected:
     NodeIteratorBase(Node&, unsigned whatToShow, RefPtr<NodeFilter>&&);
-    ExceptionOr<unsigned short> acceptNode(Node&);
+    ExceptionOr<unsigned short> acceptNode(Node& node)
+    {
+        if (!m_filter)
+            return matchesWhatToShow(node) ? NodeFilter::FILTER_ACCEPT : NodeFilter::FILTER_SKIP;
+        return acceptNodeSlowCase(Node&);
+    }
+
+    bool matchesWhatToShow(Node& node)
```

Comment:
seems like the member function could be const. The node could likely be passed as `const Node&` too.

Reply:
Good point. Will make it const.